analysis.md

```diff
@@ -0,0 +1,77 @@
+# Malware Analysis
+
+## Extracting the Source Code
+The file hash of the malicious document is `340795d1f2c2bdab1f2382188a7b5c838e0a79d3f059d2db9eb274b0205f6981`. I used [oletools](https://github.com/decalage2/oletools) to extract the VBA source code from the macro.
+
+## Layer 1
```

Comment:
How about adding a few words to describe what Layer 1 means? Like: `## Layer 1 - VB Executing PowerShell`